internal/pkg/client/cache/oras.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package cache
+
+import (
+	"os"
+	"path/filepath"
+
+	"github.com/sylabs/singularity/internal/pkg/oras"
+	"github.com/sylabs/singularity/internal/pkg/sylog"
+)
+
+const (
+	// OrasDir is the directory inside the cache.Dir where oras images are cached
+	OrasDir = "oras"
```

Comment:
does this need to be exported?

Reply:
No, I'm just following the convention of the other cache types